package.json

```diff
@@ -32,8 +32,8 @@
   },
   "scripts": {
     "prepublish": "babel src --out-dir dist",
-    "lint": "eslint .",
-    "tape": "babel-tape-runner 'src/**/__tests__/*.js'",
+    "lint": "eslint src",
```

Comment:
This is not working on windows ? I want to lint not only src, but probably some others folder like `scripts` (later)